<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>步骤1</string>
    </property>
    <widget class="QPushButton" name="button_login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;第一步，点击“登录账号”在弹出的浏览器中输入登录企查查。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>登录账号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>点击登录账号，在弹出的浏览器中登录企查查。</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>681</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>步骤2</string>
    </property>
    <widget class="QPushButton" name="button_crawl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>批量抓取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>批量抓取前，先在下面的编辑框中输入要抓取的法人名称。一个名称一行。</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_edit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>641</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>681</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>步骤3</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>若出错，关闭浏览器，重复步骤1，2。</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
